libkbfs/disk_block_cache.go

```diff
@@ -411,8 +404,22 @@ func (cache *DiskBlockCacheStandard) Put(ctx context.Context, tlfID tlf.ID,
 		return err
 	}
 	if !hasKey {
-		for cache.currBytes+uint64(encodedLen) > cache.maxBytes {
-			cache.evictLocked(ctx, defaultNumBlocksToEvict)
+		for {
```

Comment:
I think you want to limit the iteration count here, to avoid infinite loops if somehow there's a bug in the limiter, or when we don't want to put the block in the cache, even if we evict everything from the cache first. (see comment above for example)